grammar.y

```diff
@@ -518,4 +519,9 @@ Finally:
     FINALLY Block
     ;
 
-%%
\ No newline at end of file
+ThrowStatement:
+    THROW Expression
+    "identifier"
```

Comment:
remove this line. See http://www.ecma-international.org/ecma-262/6.0/index.html#sec-throw-statement